invenio_oaiserver/config.py

```diff
@@ -48,6 +48,13 @@
 OAISERVER_METADATA_FORMATS = {
     'oai_dc': {
         'serializer': 'dojson.contrib.to_marc21.utils:dumps',
+        'etree_dumper': (
```

Comment:
I was not sure if calling it `serializer` would been the best name because a `etree` object doesn't sound like a "serialized" object but more like "intermediate" structure.